app/src/main/java/com/openvehicles/OVMS/entities/CarData.kt

```diff
@@ -208,6 +216,13 @@ class CarData : Serializable {
     var car_temp_charger_raw = 0f
     var car_temp_ambient_raw = 0f
     var car_temp_cabin_raw = 0f
+    var car_booster_metrics = false
+    var car_booster_on = "no"
+    var car_booster_weekly = "no"
+    var car_booster_time = "0515"
+    var car_booster_ds = 0
+    var car_booster_de = 0
+    var car_booster_bdt = 0
```

Comment:
The Smart has a quite different meaning of "boosting" than any other vehicle would have, as it relates to the Smart's climate control / heating system. So please rename these variables to make clear they are HVAC specific, i.e. name them following the scheme `car_ac_booster_…`.